dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
+
+import java.sql.SQLException;
+
+import dbfit.fixture.StatementExecution;
+
+public interface DbStoredProcedureCall {
+
+    String getName();
+
+    boolean isFunction();
+
+    String toSqlString();
```

Comment:
@javornikolov any thoughts on, perhaps, removing the `DbStoredProcedureCall` interface and letting `buildPreparedStatement` of `DbObject` create the appropriate statement executor?